easybuild/easyconfigs/z/zlib/zlib-1.2.8-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.8'
```

Comment:
this version can probably be removed from the PR replaced by 1.2.11

Reply:
done.